README.md

```diff
@@ -44,23 +44,23 @@ m6_web_log_bridge:
         - all_error
     filters:
             get_article_error:
-                route: get_article
+                routes: ['get_article']
                 method: ['GET']
                 status: [422, 500]
                 level: 'error'
                 options:
                     response_body: true # from add Response body content (with DefaultFormatter)
             post_article_all:
-                route: post_article
+                routes: ['post_article']
                 method: ~ # from all methods
                 status: ~ # from all status
             get_article_not_found:
-                route: get_article
+                route: ['get_article']
                 method: ['GET']
                 status: [404]
                 level: 'warning'
             edit_category:
-                route: get_category
+                route: ['get_category']
```

Comment:
```suggestion
                routes: ['get_category']
```

except if you aim to keep supporting the existing `route` key (which may be a good idea... but in this case, we should not support passing an array with the `route` key so you would have to change something here anyway)

Reply:
You're totally right 😄